src/modules/src/Kconfig

```diff
@@ -59,3 +59,49 @@ config ESTIMATOR_OOT
 	default n
 
 endmenu
+
+menu "Motor configuration"
+
+config MOTORS_START_DISARMED
+    bool "Set disarmed state after boot"
+    default n
+    help
+      When enabled, the firmware will boot in disarmed state and one needs to
+      arm the drone explicitly before starting the motors
+
+config MOTORS_DEFAULT_IDLE_THRUST
+    int "Default idle thrust for motors in armed state"
+    range 0 65535
+    default 0
+    depends on MOTORS_START_DISARMED
+    help
+      Default thrust for motors when idling in armed state, expressed as an
+      integer in the range 0 to 65535. This can be overridden with parameters;
+	  the value specified here applies to the case when the persistent storage
+	  does not contain an idle thrust value.
+
+config MOTORS_DEFAULT_PROP_TEST_PWM_RATIO
+    int "Override default PWM ratio to use during motor tests"
+    range 0 65535
+    default 0
+    help
+      PWM ratio to use during motor tests, expressed as an integer in the range
+	  0 to 65535. Zero means not to override the PWM ratio; in this case, the
+	  drone uses full thrust for brushed motors and does not spin up brushless
+	  motors (for safety reasons). It is therefore advised to set this parameter
+	  explicitly for brushless builds to enable the motor test functionality.
+	  
+	  The value specified here can also be overridden with parameters.
+
+endmenu
+
+menu "Parameter subsystem"
+
+config PARAM_SILENT_UPDATES
+    bool "Silent parameter updates"
+    default n
+    help
+      Do not send CRTP packets when a parameter is updated
```

Comment:
Do we need a stronger note here? Some examples/behavior in the python lib will break if this is enabled, right? @ataffanel ? @knmcguire ?

Reply:
I think that most python examples use set_value for the parameter update, which does not wait for a callback. But I think will be a few on which this will fail yes, so this should not be on by standard for sure. 

This might be an issue for the automated testing though, but let's see. We can fix that once it emerges.